packages/core-mobile/app/services/fcm/AppCheckService.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  firebase,
+  FirebaseAppCheckTypes
+} from '@react-native-firebase/app-check'
+import Logger from 'utils/Logger'
+
+class AppCheckService {
+  init = (): void => {
+    const rnfbProvider = firebase
+      .appCheck()
+      .newReactNativeFirebaseAppCheckProvider()
+    rnfbProvider.configure({
+      android: {
+        provider: __DEV__ ? 'debug' : 'playIntegrity',
+        debugToken:
+          'some token you have configured for your project firebase web console'
+      },
+      apple: {
+        provider: __DEV__ ? 'debug' : 'appAttestWithDeviceCheckFallback',
+        debugToken: process.env.APPCHECK_DEBUG_TOKEN_APPLE
```

Comment:
why are you using process.env here instead of react native config? I assume it's not necessary to put in an env file? can you add some docs on how to debug app check with this? like the steps and so on.